### DISCUSSION TITLE
Broadcast input to multiple panels in the same terminal window

### DISCUSSION THREAD

**Discussion by faizalrf** [microsoft/terminal#15081] [Ideas]
Hello awesome folks,

I know this has been requested multiple times and all those feature requests have been closed back in 2019 and no update since. 

Any idea if there will ever be a possibility of being able to send the same command to multiple tabs or multiple planes?

This is super useful when working with multiple servers or sessions.

Cheers,
Faisal.

**Comment by j4james** [comment]
The feature requests that you've seen closed are just duplicate issues - the original issue is still open (see #2634). There is also already a PR in progress (see #14393), but that seems to have been on hold for a while. It's possible it could be in the next release though. The milestone is currently set to v1.18.
